#!/usr/bin/env python
PACKAGE = "pcl_filters"
NODE = "statisticaloutlierremoval_filter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("neighbours", int_t, 0, "Number of neighbours to take the mean", 50, 0, 100)
gen.add("standard_deviation", double_t, 0, "SD to check the deviation from the mean value", 0.5, 0, 1)
gen.add("statisticaloutlierremoval_sub", str_t, 0, "The name of the topic the statisticaloutlierremoval is subscribing", "/statisticaloutlierremoval_filter/input_cloud")

exit(gen.generate(PACKAGE, NODE, "statisticaloutlierremoval"))
